classes/Pods.php

```diff
@@ -1675,7 +1659,9 @@ public function field( $name, $single = null, $raw = false ) {
 
 									foreach ( $data as $item_id => $item ) {
 										// $field is 123x123, needs to be _src.123x123
-										$full_field = implode( '.', array_splice( $params->traverse, $key ) );
+										$traverse_fields    = array_splice( $params->traverse, $key );
+										$full_field         = implode( '.', $traverse_fields );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 9 spaces

<sub>hash: 405a27f6be2b68bf6e8e49019505b456</sub>